apps/dashboard/src/components/contract-components/contract-deploy-form/contract-metadata-fieldset.tsx

```diff
@@ -41,10 +41,7 @@ export const ContractMetadataFieldset: React.FC<
                   shouldTouch: true,
                 })
               }
-              border="1px solid"
-              borderColor="gray.200"
-              borderRadius="md"
-              transition="all 200ms ease"
+              className="border rounded transition-all duration-200"
```

Comment:
border-border to make sure the color is right when used inside pages?

Reply:
good catch, added